tests/config/format-test.js

```diff
@@ -1,30 +1,32 @@
+"use strict";
```

Comment:
why did we add this? is this not always the case?

Reply:
I want to keep these files as close as possible to prettier's so it's easier to update them following their new practices